php-practice/section1.php

```diff
@@ -144,6 +169,14 @@
 // PROBLEM 9: Remove element 2 from $myArray and then print out the array.
 // =======================================================================
 
+$myArray = ["Bob Ross", 102, 3.14, true];
+
+$myArray[4] =  "I am Russell and I eat arrays for breakfast";
+
+unset($myArray[3]);
```

Comment:
The question says remove element 2 -- the element at index `3` would be element 4.